### PR TITLE
Doc in_tail permission

### PR DESCRIPTION
**Pull request by brsolomon-deloitte**
Related:

- https://github.com/fluent/fluentd/issues/3525
- https://github.com/fluent/fluentd/pull/3478

### CODE REVIEW

File: input/tail.md [hunk 1]
```diff
@@ -417,6 +417,13 @@ If you see this message:
 
 It means that `fluentd` does not have read permission for `/path/to/file`. Check your fluentd and target files permission.
 
+**Note**: When `td-agent` is launched by systemd, the user of the `td-agent` process is the `td-agent` user.
```
Comment (cosmo0920) [inline input/tail.md]: _by default_ is needed?

the user of the `td-agent` process is the `td-agent` user _by default_.

File: parser/README.md [hunk 2]
```diff
@@ -41,6 +41,11 @@ Here is an example to read Nginx access logs using `in_tail` and `parser_nginx`:
 </source>
 ```
 
+**Note**: When `td-agent` is launched by systemd, the user of the `td-agent` process is the `td-agent` user.
```
Comment (cosmo0920) [inline parser/README.md]: ditto.

File: input/tail.md [hunk 3]
```diff
@@ -417,6 +417,13 @@ If you see this message:
...
+You must ensure that this user has read permission to the tailed `/path/to/file`. For instance, on Ubuntu,
+the default Nginx access file `/var/log/nginx/access.log` is mode `0640` and owned by `www-data:adm`. In
+this case `td-agent` must be added to the `adm` group.
+
+A bug exists in Fluentd 1.13.x where it may suppress warning logs about unreadable files. (See Fluentd PR [#3478](https://github.com/fluent/fluentd/pull/3478).)
+
```
Comment (kenhys) [inline input/tail.md:430]: Adding a link to https://docs.fluentd.org/deployment/linux-capability#capability-handling-on-in_tail may be useful.

File: input/tail.md (hunk 1 above)
Comment (brsolomon-deloitte) [reply input/tail.md]: Clarified in e6d9693

File: parser/README.md (hunk 2 above)
Comment (brsolomon-deloitte) [reply parser/README.md]: Clarified in e6d9693

File: input/tail.md (hunk 3 above)
Comment (brsolomon-deloitte) [reply input/tail.md:430]: Well I just learned something new about fluentd.

Added in 1668af1.

**Review by cosmo0920 — CHANGES_REQUESTED** [review]
commit https://github.com/fluent/fluentd-docs-gitbook/pull/370/commits/e6d969369357585ee83590b8d43836c1fb2d40b3 is not included Signed-off-by line. Could you add it there?

**Comment by brsolomon-deloitte** [comment]
@cosmo0920 done, squashed commits and signed.

**Review by cosmo0920 — APPROVED** [review]
Great. Thanks!

**Pull request merged**